acceptance/tests/peering/peering_connect_test.go

```diff
@@ -287,6 +296,70 @@ func TestPeering_Connect(t *testing.T) {
 				k8s.CheckStaticServerConnectionSuccessful(t, staticClientOpts, staticClientName, "http://localhost:1234")
 			}
 
+			// Check that requests can reach the external static-server from the static client cluster.
+			if cfg.EnableTransparentProxy {
+				const (
+					externalServerK8sNamespace = "external"
+					externalServerServiceName  = "static-server"
+					externalServerHostnameID   = "static-server-hostname"
+					terminatingGatewayRules    = `service_prefix "static-server" {policy = "write"}`
+				)
+
+				// Create the namespace for the "external" static server.
+				externalServerOpts := &terratestk8s.KubectlOptions{
+					ContextName: staticServerOpts.ContextName,
+					ConfigPath:  staticServerOpts.ConfigPath,
+					Namespace:   externalServerK8sNamespace,
+				}
+				logger.Logf(t, "creating namespace %s in server peer", externalServerK8sNamespace)
+				k8s.RunKubectl(t, staticServerPeerClusterContext.KubectlOptions(t), "create", "ns", externalServerK8sNamespace)
+				helpers.Cleanup(t, cfg.NoCleanupOnFailure, cfg.NoCleanup, func() {
+					k8s.RunKubectl(t, staticServerPeerClusterContext.KubectlOptions(t), "delete", "ns", externalServerK8sNamespace)
+				})
+
+				// Create the external server in the server Kubernetes cluster, outside the mesh in the "external" namespace
+				logger.Log(t, "creating static-server deployment in server peer outside of mesh")
+				k8s.DeployKustomize(t, externalServerOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/bases/static-server")
+
+				// Prevent dialing the server directly through the sidecar.
+				terminatinggateway.CreateMeshConfigEntry(t, staticServerPeerClient, "")
+				terminatinggateway.CreateMeshConfigEntry(t, staticClientPeerClient, "")
+
+				// Create the config entry for the terminating gateway
+				terminatinggateway.CreateTerminatingGatewayConfigEntry(t, staticServerPeerClient, "", "", externalServerHostnameID)
+				if c.ACLsEnabled {
+					// Allow the terminating gateway write access to services prefixed with "static-server".
+					terminatinggateway.UpdateTerminatingGatewayRole(t, staticServerPeerClient, terminatingGatewayRules)
+				}
+
+				// This is the URL that the static-client will use to dial the external static server in the server peer.
+				externalServerHostnameURL := fmt.Sprintf("http://%s.virtual.%s.consul", externalServerHostnameID, staticServerPeer)
+
+				// Register the external service.
+				terminatinggateway.CreateServiceDefaultDestination(t, staticServerPeerClient, "", externalServerHostnameID, "http", 80, fmt.Sprintf("%s.%s", externalServerServiceName, externalServerK8sNamespace))
+				// (t-eckert) this shouldn't be required but currently is with HTTP services. It works around a bug.
+				helpers.RegisterExternalService(t, staticServerPeerClient, "", externalServerHostnameID, fmt.Sprintf("%s.%s", externalServerServiceName, externalServerK8sNamespace), 80)
```

Comment:
No luck finding those docs after all

Reply:
>  It's kind of confusing to me that this test registers the external service in both ways. There are Consul docs somewhere that, IIRC, say Consul will prefer the node service registration in this case.

I think we could get away with just registering the service with Consul, if there was a way to send in the protocol, but [it doesn't look like there is](https://developer.hashicorp.com/consul/docs/services/configuration/services-configuration-reference).

Reply:
This was one of the things I had to work through in writing this test. I found that both registration methods were required. Service Defaults to specify the protocol and Node registration for everything else. 

I don't think this _should_ be the case and I think we have a feature gap or bug here.

Reply:
It absolutely makes sense to use `service-defaults` to set the protocol; however, we also set the `destinations` field on there which is the part that I find odd. I think it might be an artifact of re-using code in different places but I'm not sure (weekend brain)

Reply:
Yeah I think it is just from reusing the code. I do think this is fine, when testing this manually it never interfered with anything. It is something to keep in mind for when the bugs related to these feature get sorted out though. Probably will need more specific tests by then anyway.

Reply:
Some notes from offline conversation:

We know that requiring node registration is a bug, but we aren't sure if it worked properly in the past. We want to ensure that we can get to this same working system without explicitly setting the destination on the Service Default. 

For now, we need to create both in order to set the protocol using the Service Default and configure the service using node registration.